### PR TITLE
[demo-express] Update Presentation test

### PR DESCRIPTION
**Pull request by chunywang**
Impacted TCs num(approved): New 0, Update 1, Delete 0
Unit test Platform: Crosswalk Project for Android 19.48.485.0
Unit test result summary: Pass 1, Fail 0, Blocked 0

https://crosswalk-project.org/jira/browse/XWALK-5801

### CODE REVIEW

File: samples/Presentation/README.md [hunk 1]
```diff
@@ -1,3 +1,13 @@
 ## Usecase Design
 
-This sample demonstrates the external displays from within web applications.
+This sample demonstrates the external displays from within web applications, inclue:
+
+* Monitor availability of presentation displays, start a new presentation, terminate presentation and reconnect to the presentation
```
Comment (zqzhang) [inline samples/Presentation/README.md]: Why not make new lines break for each of the functions?

File: samples/Presentation/contents.html [hunk 2]
```diff
@@ -27,29 +27,36 @@
 
 Authors:
         Liu.Xin <xinx.liu@intel.com>
-
+        Wang, Chunyan <chunyanx.wang@intel.com>
```
Comment (zqzhang) [inline samples/Presentation/contents.html]: Indent

File: samples/Presentation/index.html [hunk 3]
```diff
@@ -27,7 +27,7 @@
 
 Authors:
         Liu.Xin <xinx.liu@intel.com>
-
+		    Wang, Chunyan <chunyanx.wang@intel.com>
```
Comment (zqzhang) [inline samples/Presentation/index.html]: Indent

File: samples/Presentation/index.html [hunk 4]
```diff
@@ -66,8 +76,8 @@ <h3 id="main_page_title"></h3>
         <li>Turn on ‘Wireless display’ to connect to the available wireless display.</li>
       </ol>
     </li>
-    <li>
-    Alertnative: no wireless display
+	  <li>
```
Comment (zqzhang) [inline samples/Presentation/index.html]: indent

**Comment by Honry** [comment]
@zqzhang, can we merge it now?

**Comment by zqzhang** [comment]
Yes, but I have no push access.

@jennycao

**Pull request merged**